src/marvin/beta/assistants/assistants.py

```diff
@@ -76,34 +76,33 @@ def get_instructions(self) -> str:
     @expose_sync_method("say")
     async def say_async(
         self,
-        message: str,
+        message,
         file_paths: Optional[list[str]] = None,
-        thread: Optional[Thread] = None,
-        return_user_message: bool = False,
+        thread=None,
+        event_handler_class=NOT_PROVIDED,
         **run_kwargs,
-    ) -> list[Message]:
-        """
-        A convenience method for adding a user message to the assistant's
-        default thread, running the assistant, and returning the assistant's
-        messages.
-        """
+    ):
```

Comment:
what is the type is this `return` now?

Reply:
It's a `Run` object that contains `.steps` and `.messages`. Updated.